configure.ac

```diff
@@ -6567,88 +6696,186 @@ AC_COLLECTD([werror],    [disable], [feature], [building with -Werror])
+plugin_lua="$with_liblua"
+plugin_madwifi="$have_linux_wireless_h"
+plugin_match_empty_counter="yes"
+plugin_match_hashed="yes"
+plugin_match_regex="yes"
+plugin_match_timediff="yes"
+plugin_match_value="yes"
+plugin_mbmon="yes"
 plugin_mcelog="no"
+plugin_md="$have_linux_raid_md_u_h"
 plugin_mdevents="no"
+plugin_memcachec="$with_libmemcached"
+plugin_memcached="yes"
 plugin_memory="no"
+plugin_mic="$with_mic"
 plugin_mmc="no"
+plugin_modbus="$with_libmodbus"
+plugin_mqtt="$with_libmosquitto"
 plugin_multimeter="no"
+plugin_mysql="$with_libmysql"
+plugin_netapp="$with_libnetapp"
+plugin_netlink="$with_libmnl"
 plugin_netstat_udp="no"
+plugin_network="yes"
 plugin_nfs="no"
+plugin_nginx="$with_libcurl"
+plugin_notify_desktop="$with_libnotify"
+plugin_notify_email="$with_libesmtp"
+plugin_notify_nagios="yes"
+plugin_ntpd="yes"
 plugin_numa="no"
+plugin_nut="$with_libupsclient"
+plugin_olsrd="yes"
+plugin_onewire="$with_libowcapi"
+plugin_openldap="$with_libldap"
+plugin_openvpn="yes"
+plugin_oracle="$with_oracle"
 plugin_ovs_events="no"
 plugin_ovs_stats="no"
 plugin_pcie_errors="no"
 plugin_perl="no"
+plugin_pf="$have_net_pfvar_h"
 plugin_pinba="no"
+plugin_ping="$with_liboping"
+plugin_postgresql="$with_libpq"
+plugin_powerdns="yes"
 plugin_processes="no"
 plugin_procevent="no"
 plugin_protocols="no"
 plugin_python="no"
+plugin_redfish="$with_libredfish"
+plugin_redis="$with_libhiredis"
+plugin_routeros="$with_librouteros"
+plugin_rrdcached="$librrd_rrdc_update"
+plugin_rrdtool="$with_librrd"
+plugin_sensors="$with_libsensors"
 plugin_serial="no"
+plugin_sigrok="$with_libsigrok"
+plugin_slurm="$with_libslurm"
 plugin_smart="no"
+plugin_snmp="$with_libnetsnmp"
+plugin_snmp_agent="$with_libnetsnmpagent"
+plugin_statsd="yes"
 plugin_swap="no"
 plugin_synproxy="no"
 plugin_sysevent="no"
+plugin_syslog="$have_syslog"
+plugin_table="yes"
+plugin_tail_csv="yes"
+plugin_tail="yes"
 plugin_tape="no"
+plugin_target_notification="yes"
+plugin_target_replace="yes"
+plugin_target_scale="yes"
+plugin_target_set="yes"
+plugin_target_v5upgrade="yes"
 plugin_tcpconns="no"
+plugin_teamspeak2="yes"
 plugin_ted="no"
 plugin_thermal="no"
+plugin_threshold="yes"
+plugin_tokyotyrant="$with_libtokyotyrant"
 plugin_turbostat="no"
 plugin_ubi="no"
+plugin_unixsock="yes"
 plugin_uptime="no"
 plugin_users="no"
+plugin_uuid="yes"
+plugin_varnish="$with_libvarnish"
 plugin_virt="no"
 plugin_vmem="no"
 plugin_vserver="no"
 plugin_wireless="no"
+plugin_write_graphite="yes"
+plugin_write_http="$with_libcurl"
+plugin_write_influxdb_udp="yes"
+plugin_write_kafka="$with_librdkafka"
 plugin_write_log="no"
+plugin_write_mongodb="$with_libmongoc"
 plugin_write_prometheus="no"
+plugin_write_redis="$with_libhiredis"
+plugin_write_riemann="$with_libriemann_client"
+plugin_write_sensu="yes"
 plugin_write_stackdriver="no"
+plugin_write_syslog="yes"
+plugin_write_tsdb="yes"
 plugin_xencpu="no"
 plugin_zfs_arc="no"
 plugin_zone="no"
 plugin_zookeeper="no"
+plugin_zookeeper="yes"
```

Comment:
"no", but "yes"???

And why some plugins are first declared with  `="$with_something"`, and right after that either as `="no"` or `="yes"`, but some plugins in above list have just `="$with_something"`?

Reply:
\*lol\* the additions have been generated from the arguments that have previously been passed  the `AC_PLUGIN` macro. Apparently `plugin_zookeeper="no"` was defined and never used, hence the redefinition. I'll fix that.